scipy/stats/_qmc.py

```diff
@@ -1677,6 +1680,7 @@ class PoissonDisk(QMCEngine):
     --------
     Generate a 2D sample using a `radius` of 0.2.
 
+    >>> import numpy
```

Comment:
We probably want to follow the usual `import numpy as np` convention to match the rest of this (and other) examples.